queryparams/params.go

```diff
@@ -367,11 +369,19 @@ const (
 	Postponed
 )
 
-var relTypeNames = map[ReleaseType]string{Upcoming: "type-upcoming", Published: "type-published", Cancelled: "type-cancelled", Provisional: "subtype-provisional", Confirmed: "subtype-confirmed", Postponed: "subtype-postponed", InvalidReleaseType: "Invalid"}
+var relTypeValues = map[ReleaseType]struct{ value, iCalValue string }{
```

Comment:
May I suggest renaming the fields (and functions reading them) to something like `name` and `label`?

Reply:
Can use tern 'name', as in 'name' 'iCalName', but label is inappropriate. Also methods of the same name is inappropriate - the use of String() to fulfill the stringer interface allows:
rt := Published
fmt.Printf("release type is %s", rt)

Reply:
Ooooookay